website/project/metadata/prereg-prize-test.json

```diff
@@ -0,0 +1,422 @@
+{
+    "name": "Prereg Prize",
+    "version": 1,
+    "flags": {
+        "isApproved": false,
+        "isPendingReview": false,
+        "paymentSent": false
+    },
+    "config": {
+        "requiresApproval": true,
+        "messages": {
+            "beforeSubmitForApproval": "You are about to submit your study and analysis for review. This will notify Prereg Prize Administrators who will begin to review it. You will still be able to edit this registration but must submit again. Are you sure you are ready for review?",
+            "afterSubmitForApproval": "<img src=\"https://i.imgur.com/0aCtj3b.png\"><br><p>You have successfully submitted your study and analysis plans. Next, staff from the Center for Open Science will have the plans reviewed to assure that enough detail was included. Reviewers may accept your plans or request revisions. Remember, in order to be eligible for the Prere Prize, your plans must be accepted through this review process. We will contact you within five business days with a status update. If you have any questions, please contact <a href=\"mailto:prereg@cos.io\">prereg@cos.io</a> and we will be happy to help you.</p>"
+        },
+        "fulfills": [{
+            "name": "Prereg Prize",
+            "info": "http://centerforopenscience.org/prereg/"
+        }]
+    },
+    "description": "Pre-Registration Prize (pre-data collection): You will be asked a series of questions to ensure that your sampling, design, and analysis plans are solidified prior to beginning your study. Publications that pre-register with this template may be eligible for a $1,000 prize. <a href=\"https://osf.io/x5w7h/\">See project page for complete information</a>.",
```

Comment:
@erinmayhood -- is there any way we can reword this description that doesn't include a hard link? The schema system was built before we started sanitizing our data before presentation to users and there isn't a non-hacky way to make this work.